.asf.yaml

```diff
@@ -14,6 +14,14 @@
 # limitations under the License.
 #
 
+staging:
+  profile: ~
+  whoami: main
+  foo: trigger
+
+# asf-site branch will show up at https://seatunnel.apache.org
+publish:
+  whoami:  asf-site
 github:
   features:
```

Comment:
I found out we enable issue in L28? Is expect behavior or add by mistake?

Reply:
We need to specify the release branch

Reply:
Enable branch-protection through the whoami parameter